Screen sharing leads to crash (wayland)

**Describe the bug**
Screen sharing doesn't work in wayland sessions after the electron update. This is due to long standing bug in electron: https://github.com/electron/electron/issues/37463. There is a fix on the way, but it's not yet merged unfortunately: https://github.com/electron/electron/pull/38833.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to any meeting
2. Click on window screen share
4. See error

**Expected behavior**
Screen sharing works without issues

**Desktop (please complete the following information):**
 - OS: Fedora (doesn't matter, all OSes with Wayland enabled are affected)
 - Installation package: Flatpak (doesn't matter, all packages affected) 
 - Version: 1.2.2

Same here on Fedora 38, GNOME 44.2, Wayland, teams-for-linux 1.2.3 (*1.2.4 changelog doesn't mention this issue*). Error message:

```
[ERROR] connect ECONNREFUSED ::1:80
[169917:0711/114347.927996:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.ScreenSaver.GetActive: object_path= /org/freedesktop/ScreenSaver: org.freedesktop.DBus.Error.NotSupported: This method is not implemented
[170053:0711/114349.756925:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[170053:0711/114407.550378:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[170053:0711/114408.954844:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
Speicherzugriffsfehler (Speicherabzug geschrieben)
```

--- a/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
+++ b/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
@@ -14,6 +14,13 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
+		<release version="1.2.5" date="2023-07-11">
+			<description>
+				<ul>
+					<li>Electron version upgraded to 24.6.3</li>
+				</ul>
+			</description>
+		</release>
 		<release version="1.2.4" date="2023-07-11">
 			<description>
 				<ul>

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.2.4",
+  "version": "1.2.5",
   "main": "app/index.js",
   "description": "Unofficial client for Microsoft Teams for Linux",
   "homepage": "https://github.com/IsmaelMartinez/teams-for-linux",
@@ -45,7 +45,7 @@
     "yargs": "17.7.2"
   },
   "devDependencies": {
-    "electron": "^24.6.2",
+    "electron": "^24.6.3",
     "electron-builder": "24.4.0",
     "eslint": "8.44.0",
     "yarn": "1.22.19"

